### PR TITLE
Group docs

### PR DESCRIPTION
**Pull request by peterasujan**
Documentation update for select, drop, where, sort, group, and groups.

### CODE REVIEW

**Comment by peterasujan** [comment]
``pivot`` and ``join`` are next!

File: datascience/tables.py [hunk 1]
```diff
@@ -511,12 +511,12 @@ def copy(self, *, shallow=False):
...
     def select(self, column_label_or_labels):
-        """Return a Table with selected column or columns by label.
+        """Return a Table with selected column or columns by label or index.
 
         Args:
             ``column_label_or_labels`` (string or list of strings): The header
-            names of the columns to be selected. ``column_label_or_labels`` must
-            be an existing header name.
+            names or indices of the columns to be selected. ``column_label_or_labels`` must
+            be an existing header name, or a valid column index.
```
Comment (SamLau95) [inline datascience/tables.py]: Good point with the index behavior. Could you add an example of that below?

File: datascience/tables.py [hunk 2]
```diff
@@ -557,19 +557,56 @@ def exclude(self):
...
+
+        Args:
+            ``column_or_label``: The header name of a column in the table or an array.
+
+            ``value``: Value for comparison with items in ``column_or_label``.
+
+        Returns:
+            An instance of ``Table`` containing rows for which the ``column_or_label``
+            column or ``column_or_label`` itself is non-zero or True, or is equal to ``value``,
+            if provided.
```
Comment (SamLau95) [inline datascience/tables.py]: Awesome! While we at it, could you write an example or two for this?

**Comment by SamLau95** [comment]
Hooray!

**Pull request merged**